packages/core/src/Actor.ts

```diff
@@ -154,7 +145,7 @@ export class Actor<TEvent extends EventObject, TEmitted> {
   public current: TEmitted;
   private context: ActorContext;
   private behavior: Behavior<TEvent, TEmitted>;
-  private mailbox: TEvent[] = [];
+  private mailbox: Array<TEvent | LifecycleSignal> = [];
```

Comment:
I don't think that `LifecycleSignal` can ever hit the mailbox - those are always directly passed to `receive` in start/stop methods

```suggestion
  private mailbox: Array<TEvent> = [];
```